### ISSUE TITLE
Difference between this extension and VSCode native feature?

### ISSUE THREAD

**Issue by tranzystorek-io** [CoenraadS/Bracket-Pair-Colorizer-2#450]
Since 1.60 vscode has native bracket coloring, how is it different from this extension apart from performance improvement?

**Comment by mystiquewolf** [comment]
I propose to list also the VS Code native feature as an alternative.

**Comment by schlopp96** [comment]
> Since 1.60 vscode has native bracket coloring, how is it different from this extension apart from performance improvement?

Was literally going to post this exact same thing; I would also like to know...

**Comment by tranzystorek-io** [comment]
Seems like native is missing some features at the very least: https://github.com/CoenraadS/Bracket-Pair-Colorizer-2/issues/445#issuecomment-912559393

**Comment by Knufle** [comment]
Hi, I've been using Bracket Pair Colorizer forever but today I decided to try the native VS Code version, I noticed that for some reason, the colors are not the same and idk how to configure it so it uses the same color pattern as Bracket Pair Colorizer does, here are some screenshots:

Bracket Pair Colorizer 2 with One Dark Pro theme:
![image](https://user-images.githubusercontent.com/25941471/157313947-f3c63fd8-abfa-4914-a856-588dc90fe4d8.png)

Native Bracket Pair Colorizer with One Dark Pro theme:
![image](https://user-images.githubusercontent.com/25941471/157314069-429f7470-fbdc-42fd-bba0-0f5ec9026107.png)

Notice the square brackets inside useState for example.

Does anyone know how I could fix that? I just am so used to the previous colors that I didn't want them to change.
